<launch>
    <!-- Behavior Cloning pretraining from recorded demonstrations -->
    <arg name="demos_dir" default="$(find td3_rl_controller_high_buffer_size_respawn)/demos" />
    <arg name="model_path" default="/tmp/td3_model.pth" />
    <node name="bc_pretrain" pkg="td3_rl_controller_high_buffer_size_respawn" type="bc_pretrain.py" output="screen">
        <param name="~demos_dir" value="$(arg demos_dir)" />
        <param name="~model_path" value="$(arg model_path)" />
        <!-- original epoch was 10 -->
        <param name="~epochs" value="20" /> 
        <!-- original batch size was 512 -->
        <param name="~batch_size" value="64" />
        <param name="~lr" value="0.0003" />
        <param name="~max_action" value="0.22" />
    </node>
</launch>
